server-adapters/resteasy-undertow-cdi/src/main/java/dev/resteasy/embedded/server/UndertowBuilderConfigurer.java

```diff
@@ -0,0 +1,11 @@
+package dev.resteasy.embedded.server;
+
+import io.undertow.Undertow;
+
+/**
+ * This interface is discovered as a ServiceLoader for configuring the {@link Undertow.Builder}
+ * before it builds the {@link Undertow} and starts it.
+ */
+public interface UndertowBuilderConfigurer {
```

Comment:
We should rename this to `UndertowBuilderConfiguator`.

Reply:
typo? how'bout `UndertowBuilderModifier`

Reply:
or `UndertowBuilderConfigurator`

Reply:
I went with your suggestion (plus the `r`)